flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/LocalExecutor.java

```diff
@@ -200,69 +277,98 @@ public void start() {
 	}
 
 	@Override
-	public List<String> listCatalogs(SessionContext session) throws SqlExecutionException {
-		final ExecutionContext<?> context = getOrCreateExecutionContext(session);
+	public void resetSessionProperties(String sessionId) throws SqlExecutionException {
+		// Renew the ExecutionContext by using the default environment.
+		this.contextMap.put(sessionId, createExecutionContext(defaultEnvironment));
```

Comment:
reset session property should keep the content which read from session environment file? Now it looks we also git rid of this.

Reply:
Yeah,  I noticed here.  I planed to not saving the session environment file in ExecutionContext before, so here seems no method to reset back to the session env.   Let me re-think about the  design , maybe we still need the session env..